crates/solver/src/solver/http_solver.rs

```diff
@@ -445,6 +445,25 @@ fn compute_fee_connected_tokens(liquidity: &[Liquidity], native_token: H160) ->
     fee_connected_tokens
 }
 
+fn non_bufferable_tokens_used(
+    interactions: &[InteractionData],
+    market_makable_token_list: &HashSet<H160>,
+) -> Vec<H160> {
+    interactions
```

Comment:
This can return some addresses multiple times. Not a problem but could avoid it by collecting into a set.